### PR TITLE
Add commandline option for labels

### LINKED ISSUE #29: Specify labels via a command-line option (rather than hardcoding)
Currently, we hardcode the "PR category" labels for projects in the source code for `changelog`:  

https://github.com/neuropoly/changelog/blob/cfd5baac53b147be2b0ecaa3997fcc1b945b9fc6/changelog/changelog.py#L303-L321

Doing it this way forces each project to open up a PR if they want to modify how their changelog is generated. (e.g. https://github.com/neuropoly/changelog/pull/28). 

Instead, perhaps we could have the usage look like this:

```
changelog --repo-url ivadomed/ivadomed --labels feature bug installation documentation enhancement [...]
```

That way, `changelog` stays project-agnostic, and each project doesn't have to open a PR every time they make a change to their label scheme. :sweat_smile:

### PR DESCRIPTION
**Pull request by cakester**
adding commandline option to use custom labels for the changelog, if commandline option is not specified, then it would use the hard coded default ones

resolves #29

### CODE REVIEW

**Comment by cakester** [comment]
@dyt811 @joshuacwnewton review plz, thanks, I don't have access to the repo so cannot assign reviewers

File: changelog/changelog.py [hunk 1]
```diff
@@ -220,6 +220,11 @@ def get_parser():
                           help="Update an existing changelog file by prepending to it.",
                           )
 
+    optional.add_argument("--labels",
+                          nargs="+",
+                          help="Labels to update to the changelog",
```
Comment (joshuacwnewton) [inline changelog/changelog.py]: Nitpick: Slight clarification about how the labels are used

```suggestion
                          help="Labels to use for grouping PRs by category (the specified labels will be used "
                               "as headers in the changelog)",
```

**Review by joshuacwnewton — COMMENTED** [review]
I like this! It feels like a pretty harmless change, since usage is optional, and we're preserving the existing behaviour by default.

I think that long-term, though, it might be good to get rid of the `get_custom_options()` and `options` snippets, then make `--labels` mandatory. That would make `changelog` a bit lighter/easier to maintain. 

But, I think it's fine to leave that until after each project tries out the `--labels` option in their workflows.

**Review by joshuacwnewton — APPROVED** [review]

**Comment by cakester** [comment]
mind merge this? I don' have write access, thanks

**Pull request merged**